interface/main/tabs/main.php

```diff
@@ -1,5 +1,6 @@
 <?php
-Use Esign\Api;
+use Esign\Api;
+use OpenEMR\Core\Header;
```

Comment:
btw, I'll leave this decision up to you (i think PSR recs to put Use at top so guessing that the route we're gonna go).